.github/workflows/stats.yml

```diff
@@ -4,12 +4,20 @@ on:
   pull_request:
     types: [opened]
 
+permissions:
+  actions: write # Required to post stats as comments.
+  contents: read # Default permission for checking out code
+
 jobs:
   stats:
     runs-on: ubuntu-latest
-    permissions:
-      contents: read
-      pull-requests: write
+
+    # This condition checks for pull requests from authors with write access.
```

Comment:
Since we know this doesn't work for PRs opened from forks, is there a way to detect that as well?

Reply:
cool updated!